ako-operator/controllers/utils.go

```diff
@@ -61,12 +61,15 @@ const (
 	TenantName             = "tenantName"
 	NoPGForSni             = "noPGForSni"
 	NsxtT1LR               = "nsxtT1LR"
+	PrimaryInstance        = "primaryInstance"
+	IstioEnabled           = "istioEnabled"
+	BlockedNamespaceList   = "blockedNamespaceList"
+	IPFamily               = "ipFamily"
+	EnableMCI              = "enableMCI"
 )
 
 var SecretEnvVars = map[string]string{
```

Comment:
@DixitAakash: Do we need to populate `certificateAuthorityData` to environment variable? AKO reads data from secret and then populate these fields.

Reply:
@DixitAakash: We will require certificateAuthorityData to environment. So please ignore this message.

Reply:
This is inline with the ako statefulset in ako helm chart. The ako statefulset will have **CTRL_CA_DATA** env variable, if **avi-secret** secret is created with **certificateAuthorityData** as one of its data.